ILAsmVersion.txt

```diff
@@ -0,0 +1 @@
+2.1.0-preview1-25916-01
```

Comment:
You should consider adding this to the auto-update list in dependencies.props. Follow the similar pattern that we use for updating the BuildToolsVersion.txt file at https://github.com/dotnet/coreclr/blob/master/dependencies.props#L97.